pallets/gear/src/queue.rs

```diff
@@ -85,17 +85,7 @@ where
                 );
             }
 
-            let (dispatch, gas, _) = precharged_dispatch.into_parts();
-            return vec![
```

Comment:
as I can guess, move of transferring value back inside core-processor was due to expectations for similar behaviour this place?
it's incorrect, message should be gone after this branch